contributor/02-becoming-a-contributor-article.md

```diff
@@ -1,13 +1,13 @@
 
-## Becoming an InnerSource Contributor
+# Becoming an InnerSource Contributor
 
 InnerSource contributors operate outside of regular team boundaries, they are the links crossing silos within organisations. As such they need to be aware of a few common practices that make this work more effective.
 
-### Sharing Mindset
+## Sharing Mindset
```

Comment:
If we're parting from using title case (which the other non-header edits aim at) then this should also be corrected.
```suggestion
## Sharing mindset
```